File: src/php/MathBenchmark.php
<?php

namespace EntityFX\NetBenchmark\Core\Generic {

    use EntityFX\NetBenchmark\Core\Writer;
    
    class MathBenchmark extends MathBase {

        protected function BenchImplementation()
        {
            $R = 0;
            $li = 0.0;

            for ($i = 0; $i < $this->Iterrations; $i++)
            {
                $R += self::DoMath($i);
            }
            return $R;
        }
    }
}

File: src/python/entityfx/math_benchmark.py
from entityfx.benchmark_base import BenchmarkBase
from entityfx.math_base import MathBase


class MathBenchmark(MathBase):
    
    def benchImplementation(self) -> float:
        r = 0
        li = 0
        i = 0
        while i < self._iterrations: 
            r += MathBase._doMath(i, li)
            li = (i); i += 1
        return r